### DISCUSSION TITLE
Apache Guacamole Documentation Question

### DISCUSSION THREAD

**Discussion by WoosterInitiative** [authelia/authelia#6018] [Troubleshooting] [answered]
I'm trying to get the Apache Guacamole OpenID integration working and have a question about the documentation.

Under the [Application](https://www.authelia.com/integration/openid-connect/apache-guacamole/#application) heading the `openid-authorization-endpoint` value includes the query parameter `state=1234abcedfdhf`; what is this? Should it be left as-is or should it be a value specific to the installation?

As written the implication is that it is static, but I have no idea what it is or what it does.

Thanks for the help!

**Comment by james-d-elliott** [comment, marked as answer]
It should be written as is, or a random string. Guacamole doesn't follow the standard and omits this parameter which we require. It's a workaround for an area it lacks.

**Reply by WoosterInitiative** [reply]
Ah. Can I suggest a note about that in the documentation?

Still struggling with Guacamole, but I'm moving that down the queue for now.

Thanks for the answer!
